records_versioning/signals.py

```
@@ -0,0 +1,92 @@
+    # If no user found, use system user if configured
+    if user is None and hasattr(settings, 'SYSTEM_USER_ID'):
+        try:
+            user = User.objects.get(id=settings.SYSTEM_USER_ID)
+        except User.DoesNotExist:
+            pass
+    
+    save_with_version(instance, user, reason='auto-signal')
+
+# Patient signal
+@receiver(post_save, sender='patients_core.Patient')
+def patient_saved(sender, instance, created, **kwargs):
```

Ruff (0.12.2)

Missing return type annotation for public function `patient_saved`

Add return type annotation: `None`

(ANN201)